mail_improved_tracking_value/models/mail_tracking_value.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import json
+
+from odoo import models, fields, api
+
+
+class MailTrackingValue(models.Model):
+
+    _inherit = "mail.tracking.value"
+
+    new_value_formated = fields.Char(compute='_compute_formatted_value',
+                                     string='New value')
+    old_value_formated = fields.Char(compute='_compute_formatted_value',
+                                     string='Old value')
+    record_name = fields.Char(related='mail_message_id.record_name')
+    model = fields.Char(related='mail_message_id.model', store='True',
+                        string='Model')
+
+    @api.depends('new_value_char', 'new_value_integer', 'new_value_float',
+                 'new_value_text', 'new_value_datetime', 'new_value_monetary',
+                 'old_value_char', 'old_value_integer', 'old_value_float',
+                 'old_value_text', 'old_value_datetime', 'old_value_monetary')
+    def _compute_formatted_value(self):
+        """ Sets the value formatted field used in the view """
+        for record in self:
+            if record.field_type in ('many2many', 'one2many', 'char'):
```

Comment:
yes, is not required, I'm just saying that since the formatted value is always a char we can format it using qweb rendering (especially for date, datetime, monetary, etc)